fbw-a32nx/src/systems/fmgc/src/flightplanning/FlightPlanManager.ts

```diff
@@ -753,7 +755,7 @@ export class FlightPlanManager {
      * @param callback A callback to call once the operation completes.
      */
     public async setDestination(icao: string, callback = () => { }): Promise<void> {
-        const sameAirport = this.getDestination()?.ident === icao;
+        const sameAirport = this.setDestination()?.ident === icao;
```

Comment:
> Reverted the bool check and it can be fixed in another PR, the scope of this PR was only to fix the API issue,

I believe the above was changed mistakenly? Doesn't make sense to compare a value against a setter

Reply:
Address before testing

Reply:
Yes was a mistake, reverted this